packages/playground/RunHeadlessJsTask.ps1

```diff
@@ -0,0 +1,54 @@
+# The 
+$packageIdentity = "59b7e31b-a8fe-4fc7-85d6-a0a2475ad32d"
+$taskName = "SampleHeadlessTask"
+
+$packages = plmdebug /query
+$packageFullName = ""
+
+if ($LASTEXITCODE -ne 0) {
```

Comment:
>$LASTEXITCODE [](start = 4, length = 13)

I guess you checked the wrong lastexitcode, and it's possible

$packageFullName = ""
$packages = plmdebug /query